components/fieldgroups/experience-event/experienceevent-botdetection.schema.json

```diff
@@ -0,0 +1,38 @@
+{
+  "meta:license": [
+    "Copyright 2023 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/experienceevent-botdetection",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Bot Detection Information",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:tags": {
+    "industry": ["all"]
+  },
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Provides information about bot generated traffic.",
+  "definitions": {
+    "experienceevent-botdetection": {
+      "properties": {
+        "xdm:botDetection": {
+          "title": "Bot Detection",
+          "$ref": "https://ns.adobe.com/xdm/context/botdetection",
```

Comment:
If bot detection is only going to be used here, let's not create any data type, just define the properties inline here